tests/testthat/_snaps/transformer.md

```diff
@@ -0,0 +1,19 @@
+# get nice errors if rlang installed
+
+    Code
+      identity_transformer("x + ")
+    Condition
+      Error:
+      ! Failed to parse glue component
+      Caused by error in `parse()`:
+      ! <text>:2:0: unexpected end of input
+      1: x + 
+         ^
+    Code
+      identity_transformer("NOTFOUND")
+    Condition
+      Error:
+      ! Failed to evaluate glue component {NOTFOUND}
+      Caused by error in `identity_transformer()`:
+      ! argument "envir" is missing, with no default
```

Comment:
This doesn't seem to be the error we're trying to test.

After staring long and hard at the CI failures, I realized this. I was puzzled why the R version mattered, since we're using `rlang::abort()`. But this error is coming from base R.